app.json

```diff
@@ -5,30 +5,28 @@
     "version": "1.0.0",
     "orientation": "portrait",
     "icon": "./assets/icon.png",
+    "userInterfaceStyle": "light",
     "splash": {
       "image": "./assets/splash.png",
       "resizeMode": "contain",
       "backgroundColor": "#ffffff"
     },
-    "updates": {
-      "fallbackToCacheTimeout": 0
-    },
-    "assetBundlePatterns": [
-      "**/*"
-    ],
+    "assetBundlePatterns": ["**/*"],
     "ios": {
       "supportsTablet": true,
-      "bundleIdentifier": "stytch.rn.test"
+      "bundleIdentifier": "com.stytch.sdk.expo.example"
     },
     "android": {
       "adaptiveIcon": {
         "foregroundImage": "./assets/adaptive-icon.png",
-        "backgroundColor": "#FFFFFF"
+        "backgroundColor": "#ffffff"
       },
-      "package": "stytch.rn.test"
+      "package": "com.stytch.sdk.expo.example"
     },
     "web": {
       "favicon": "./assets/favicon.png"
-    }
+    },
+    "owner": "stytch",
```

Comment:
Same here, I think this owner should be set depending upon the account they used for `eas login`.